docs/gui/menus.md

```diff
@@ -273,12 +273,19 @@ A `MenuProvider` is an interface that contains two methods: `#createMenu`, which
 A `MenuProvider` can easily be created using `SimpleMenuProvider`, which takes in a method reference to create the server menu and the title of the menu.
 
 ```java
-// In some implementation
-NetworkHooks.openScreen(serverPlayer, new SimpleMenuProvider(
+/*
+  In some implementation
+  args:
+   IForgeServerPlayer pPlayer , BlockPos pPos
+  use:
+  {@link IForgeServerPlayer#openMenu(MenuProvider, BlockPos)}
+  {@link IForgeServerPlayer#openMenu(MenuProvider, Consumer)}
```

Comment:
Remove this. The arguments are wrong and doesn't tell me when I should use the position versus the consumer. I assume the consumer actually contains a buffer to write arbitrary data to, but that is explained above. You can add another example there if you wish, otherwise provide the parameter.